packages/bitstring/bitstring.2.1.0/opam

```diff
@@ -31,7 +31,7 @@ You can use this module to both parse and generate binary formats, files and pro
 Bitstring handling is added as primitives to the language, making it exceptionally simple to use and very powerful."""
 flags: light-uninstall
 url {
-  src: "https://bitbucket.org/thanatonauts/bitstring/get/v2.1.0.tar.gz"
+  src: "https://github.com/xguerin/bitstring/archive/v2.1.0.tar.gz"
```

Comment:
@mseri I was not able to find this source in any of the archives using the recipes in https://github.com/ocaml/opam-repository/wiki/FAQ#how-to-find-lost-archives-of-packages

E.g., I've tried

- https://opam.robur.coop/cache/sha256/a8/a848f15787709e88bf97a81d5e7dc0115ad83b3cf36bbaec6a69fdf3d102f7bd
- https://opam.ocaml.org/cache/sha256/a8/a848f15787709e88bf97a81d5e7dc0115ad83b3cf36bbaec6a69fdf3d102f7bd
- https://opam.ocaml.org/cache/md5/70/70f7acebe337072bff03a1f7a3fb1d3a

Do we have any other option (or might I be making a mistake with with the construction of the URLs)? Or will we need to accept a changed checksum if this is to remain available?